### DISCUSSION TITLE
[Suggestion] More item data

### DISCUSSION THREAD

**Discussion by DarthGigi** [HypixelDev/PublicAPI#638] [Ideas]
It would be nice to have more data returned from the api about a skyblock item, for example, the attributes an item has.
![image](https://github.com/HypixelDev/PublicAPI/assets/47110839/b42e5f55-6811-402f-bfbc-1ca2eb092d99)
(screenshot from the hypixel skyblock wiki)

**Comment by ConnorLinfoot** [comment]
The items data is provided via our internal definitions of items, anything that isn't part of that endpoint already isn't something easily accessible to the API as it's likely only defined in code

**Reply by DarthGigi** [reply]
Thanks for your reply, after further digging and asking the community I learned that the API actually does return this data, but kind of in a weird way:

> Sellable: "npc_sell_price" (exists or does not exist)
> Tradable: "can_trade", "soulbound"
> Auctionable: "can_auction", "soulbound"
> Reforgeable: "cannot_reforge", "category"? (for items in categories that can be reforged)
> Enchantable: "category"? (for items in the enchantable category)
> Museum: "museum" (probably only for items that can be included in Special Items)

However I noticed that some items do not contain of these keys in their response when they should, for example, Minions do not contain `can_auction` (which means they are auctionable, because in other unauctionable items this key exist and has the value set to `false`).
Are there more inconsistencies / items that do not return a key when they should?

Also, I couldn't find anything about these keys in the API documentation, shouldn't these be mentioned somewhere there?
